src/views/ManageList.tsx

```diff
@@ -20,10 +22,13 @@ const purchaseTimelines = {
 };
 
 export function ManageList({ listPath }: Props) {
+	const { user: currentUser } = useAuth();
```

Comment:
question: Was there a reason to call `useAuth()` here? I know the user is needed on [line 110](https://github.com/the-collab-lab/tcl-77-smart-shopping-list/blob/rc-ma-share-shopping-list/src/views/ManageList.tsx?plain=1#L110) and I'm just curious if it was considered adding the `user` prop to the `<ManageList />` in `App.tsx` and add it to the props in the call here.

Reply:
It was not! That's all we could think to do since currentUser had to come from somewhere.